lib/helpers.js

```diff
@@ -46,29 +46,20 @@ function normalizeFileList (fileList, projectLocation) {
   return result;
 }
 
-// TODO: make this a promise with the util.promisify
-function createDir (directoryToCreate) {
-  return new Promise((resolve, reject) => {
-    mkdirp(directoryToCreate, (err) => {
-      if (err) {
-        return reject(new Error(err));
-      }
-
-      return resolve();
-    });
-  });
+async function createDir (directoryToCreate) {
+  try {
+    await promisify(mkdirp)(directoryToCreate);
```

Comment:
I think you can do this in one line, `return promisify(mkdirp)(directoryToCreate)` instead of using the async/await stuff

No need to Reject with an error since the promisify utility will already do that


Same thing for the rimraf code

Reply:
Sounds good, I did this way to pass the tests. I'll try what you suggesting to see what happen

Reply:
one line working fine, thanks for the code review!